AnkiDroid/src/main/java/com/ichi2/anki/dialogs/customstudy/CustomStudyDialog.kt

```diff
@@ -445,7 +446,7 @@ class CustomStudyDialog(private val collection: Collection, private val customSt
             Timber.i("Found deck: '%s'", customStudyDeck)
             if (cur.isStd) {
                 Timber.w("Deck: '%s' was non-dynamic", customStudyDeck)
-                showThemedToast(requireContext(), getString(R.string.custom_study_deck_exists), true)
+                showSnackbar(getString(R.string.custom_study_deck_exists), Snackbar.LENGTH_SHORT)
```

Comment:
You don't have to call `getString` for simple strings. Didn't run this change but I think it should work

```suggestion
                showSnackbar(R.string.custom_study_deck_exists, Snackbar.LENGTH_SHORT)
```

Same in other places